src/smart_qq_bot/http_client.py

```diff
@@ -24,8 +24,8 @@ class HttpClient(object):
     # urllib2.install_opener(_req)
 
     def __init__(self, load_cookie=False, cookie_file=COOKIE_FILE):
-        if not os.path.isdir("./cookie"):
-            os.mkdir("./cookie")
+        if not os.path.isdir(os.path.dirname(cookie_file)):
```

Comment:
Have you tested this feature yet?
If tested, we can merge it into the master right now:)

@Yinzo Any other problem?